src/include/common/rpp/handle.hpp

```diff
@@ -93,11 +89,9 @@ struct Handle : rppHandle
     void SetAllocator(rppAllocatorFunction allocator, rppDeallocatorFunction deallocator, void* allocatorContext) const;
 
     // Device handle related
-    Handle(rppAcceleratorQueue_t stream);
-    Handle(rppAcceleratorQueue_t stream, size_t nBatchSize);
+    Handle(size_t nBatchSize, rppAcceleratorQueue_t stream);
```

Comment:
why the parameters are reversed here? This will need to change all calling functions to make corresponding change